lib/Test/Nginx/Util.pm

```diff
@@ -2364,6 +2367,28 @@ retry:
             #warn "pid file not found";
         }
     }
+
+    if (defined $block->shutdown_error_log) {
```

Comment:
I'm so stupid, your way is more better than mine.  Please take a review on current version.